### PR TITLE
updated docs

### PR DESCRIPTION
**Pull request by CrystalOnScript**
resolves issue [3301](https://github.com/ampproject/amp.dev/issues/3301)

### CODE REVIEW

**Comment by Gregable** [comment]
I'm very confused as to why the validator code is changing here.

**Comment by CrystalOnScript** [comment]
@Gregable currently the validator throws an error when the `args` attribute it used on `amp-bind-macro`. The `argument` attribute doesn't work.

**Comment by Gregable** [comment]
To clarify for future-self reading this:

- [amp-bind-macro ](https://amp.dev/documentation/components/amp-bind/#defining-macros-with-amp-bind-macro)'s documentation uses the 'arguments' attribute.
- The validator accepts the `arguments` attribute, for example this is valid:

```
<!doctype html>
<html ⚡>
<head>
  <meta charset="utf-8">
  <link rel="canonical" href="self.html" />
  <meta name="viewport" content="width=device-width,minimum-scale=1">
  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
  <script async custom-element="amp-bind" src="https://cdn.ampproject.org/v0/amp-bind-0.1.js"></script>
</head>
<body>
  
  <amp-bind-macro
  id="circleArea"
  arguments="radius"
  expression="3.14 * radius * radius"
></amp-bind-macro>
  
  </body>
</html>
```

- The implementation of `amp-bind-macro` inspects for the `args` attribute, and any usage of `arguments` is ignored.
- Neither the documentation (currently) nor the validator refers to the `args` attribute.

Have we determined if existing documents use `arguments` yet? My concern is that there may now be documents using it. Considering that the documentation and the validator both use of `arguments`, I would imagine that most implementations would as well. 0% of valid amp implementations today use `args` by definition.

Have we considered changing the javascript implementation to match the documentation and validator instead? It seems a lower risk change.

**Comment by CrystalOnScript** [comment]
Thanks for the feedback @Gregable - I attempted to fix the runtime, but I'm unsure how to check for how many documents are currently using it. Maybe @choumx or @jridgewell would be willing to help me out?

File: extensions/amp-bind/0.1/bind-macro.js [hunk 1]
```diff
@@ -37,14 +37,14 @@ export class BindMacro {
...
   /**
    * @param {!JsonObject} scope
-   * @param {!Array} args
+   * @param {!Array} arguments
    * @throws {Error} On illegal function invocation.
    * @return {BindExpressionResultDef}
    */
-  evaluate(scope, args) {
-    const copy = /** @type {!JsonObject} */ ({...scope});
+  evaluate(scope, arguments) {
```
Comment (jridgewell) [inline extensions/amp-bind/0.1/bind-macro.js]: Naming a variable `arguments` is forbidden in strict mode, which will be a problem for module builds.

**Comment by CrystalOnScript** [comment]
@jridgewell you're right, after further investigation, the sample code needs updating - runtime and validator are fine.

File: extensions/amp-bind/0.1/bind-macro.js (hunk 1 above)
Comment (choumx) [reply extensions/amp-bind/0.1/bind-macro.js]: We should just run strict mode by default. #19380

**Pull request closed without merging**